app/my_jss/accounts/Users.js

```diff
@@ -0,0 +1,60 @@
+var jsonUsers = JSON.parse(usersJSON);
+
+function Users(){
+
+ this.allUsers = this.getAllUsers();
+};
+
+// returns boolean
+Users.prototype.checkedByType = function(type,objectKey){
+
+  var bool = false;
```

Comment:
book is a terrible nam, doesn't give any indication about what it means A good name is something like. isMatchingUser or so